src/tooltip/tooltip.ts

```diff
@@ -210,4 +216,21 @@ export class NgbTooltip implements OnInit, OnDestroy {
     }
     this._zoneSubscription.unsubscribe();
   }
+
+  private _unsubscribeScroll() {
+    if (this._scrollSubscription) {
+      this._scrollSubscription();
+      this._scrollSubscription = null;
+    }
+  }
+
+  private _subscribeScroll() {
+    this._ngZone.runOutsideAngular(() => {
+      this._scrollSubscription = subscribeScroll(this._elementRef.nativeElement, () => {
+        positionElements(
```

Comment:
And I guess we should close it, not reposition, so it really would be like `ngbAutoClose`